Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,299 @@
+        ))
+
+    return command_results
+
+
+def file_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'file/report/'
+    files = argToList(args.get('file'))
+    for file_hash in files:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'resource': file_hash})
+        res['value'] = file_hash
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            file_hash, DBotScoreType.FILE, VENDOR, score, reliability=client.reliability)
+        file_object = Common.File(dbot, md5=res.get('md5'), sha1=res.get('sha1'))
+
+        markdown = f"Threat crowd report for File {file_hash}: \n Reputation: {score_str} \n"
+        markdown += tableToMarkdown('Results', res)
+
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.File',
+            outputs=res.copy(),
+            outputs_key_field='value',
+            indicator=file_object,
+            readable_output=markdown,
+            raw_response=res
+        ))
+
+    return command_results
+
+
+def test_module(client: Client) -> str:
+    try:
+        api_url = 'ip/report/'
+        client._http_request(method='GET', url_suffix=api_url, params={'ip': '1.1.1.1'})
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set.'
+        else:
+            raise e
+    return message
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    command_functions = {'email': email_command,
+                         'domain': domain_command,
+                         'ip': ip_command,
+                         'threat-crowd-antivirus': antivirus_command,
+                         'file': file_command}
+    command = demisto.command()
+
+    demisto.debug(f'Command being called is {command}')
+    try:
+        params = demisto.params()
+        base_url = params.get('server_url')
+        verify_certificate = not params.get('insecure', False)
+        proxy = params.get('proxy', False)
+        extended_data = argToBoolean(params.get('extended_data', False))
+
+        reliability = params.get('integrationReliability')
+        reliability = reliability if reliability else DBotScoreReliability.C
+
+        if DBotScoreReliability.is_valid_type(reliability):
+            reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+        else:
+            raise Exception("Please provide a valid value for the Source Reliability parameter.")
+
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxy=proxy,
+            reliability=reliability,
+            extended_data=extended_data
+        )
```

Comment:
Is there a reason in your opinion not to extract this to a helper function called `create_client` or something similar?